### PR TITLE
docs(config): add use cache

### PR DESCRIPTION
**Pull request by Sikora00**
refs nestjs/config#121

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: nestjs/config#121

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: content/techniques/configuration.md [hunk 1]
```diff
@@ -282,6 +282,16 @@ constructor(
 
 > info **Hint** The `ConfigType` is exported from the `@nestjs/config` package.
 
+#### Use cache
+
+Accessing `process.env` is slow. To increase the performance of the `ConfigService.get` method for variables stored in the `proccess.env` set the `cache` property of the options object we pass to the `ConfigModule.forRoot()`.
```
Comment (jmcdo29) [inline content/techniques/configuration.md:287]:
```suggestion
As accessing `process.env` can be slow, you can set the `cache` property of the options object passed to `ConfigModule.forRoot()` to increase the performance of `ConfifService.get` when it comes to variables stored in `process.env`
```

**Review by jmcdo29 — CHANGES_REQUESTED** [review]
Thanks for the addition. I've added a suggestion that I think reads a little better

**Comment by Sikora00** [comment]
Greate @jmcdo29 ! Changes Applied :)

**Review by jmcdo29 — APPROVED** [review]

**Comment by kamilmysliwiec** [comment]
Thanks @Sikora00!

**Pull request merged**